content/company-info-and-process/communication/content_guidelines/style_and_mechanics.md

```diff
@@ -784,15 +784,15 @@ We describe ourselves with a few different names depending on context, and we sh
 
 - **Sourcegraph**: Main product. This name is always preferred unless you need to clarify between the 3 deployment methods for Sourcegraph below.
 - **Sourcegraph self-hosted**: On-premises and self-managed version of Sourcegraph.
-- **Sourcegraph Cloud**: This is the cloud instance of Sourcegraph at [sourcegraph.com](https://sourcegraph.com), used both for open source search as well as individual accounts for connecting public or private code.
-- **Managed Sourcegraph instance**: Private Sourcegraph instances managed and provisioned by the Sourcegraph team.
+- **Sourcegraph Cloud** Dedicated, single-tenant Sourcegraph instances managed and provisioned by the Sourcegraph team. This was previously referred to as "managed instances."
+- **Sourcegraph.com / "dotcom"**: This is the cloud / SaaS version of Sourcegraph at [sourcegraph.com](https://sourcegraph.com). It can be used freely for open source search.
```

Comment:
@iskyOS - is it confusing to refer to .com as the Cloud/SaaS version of Sourcegraph? Maybe we should move away from Cloud and focus on what you can do with .com:

Sourcegraph.com can be used to search top open source repositories.

Reply:
```suggestion
- **Sourcegraph.com / "dotcom"**: This is the service publicly available at [sourcegraph.com](https://sourcegraph.com). It can be used to search top open source repositories.
```